crates/python/qcs_sdk/qvm/__init__.pyi

```diff
@@ -25,6 +25,30 @@ class QVMResultData:
         """
         ...
 
+@final
+class QVMOptions:
```

Comment:
Other docs suggest a `.default()` method exists on this class, but does not appear here.